<launch>
    <arg name="file" default="$(env URIAL_HOME)/bags/empty.json" />
    <arg name="loop" default="0" />

    <node name="goal_manager" pkg="capra_ai" type="goal_manager.py" output="screen">
        <param name="loop" value="$(arg loop)" />
    </node>
    <node name="goal_loader" pkg="capra_ai" type="goal_loader.py" output="screen">
        <param name="file" value="$(arg file)" />
    </node>
</launch>
